### PR TITLE
US-CA-Ventura

### PR DESCRIPTION
**Pull request by ManuBenito259**
US-CA-Ventura regions and its feed us-vctc

### CODE REVIEW

**Review by damianVarela — APPROVED** [review]
fine by me as long as the validator doesn't fail.

**Comment by ManuBenito259** [comment]
I had forgotten this one thinking it had no feeds, will look for other available feeds on the area but opening the pr to begin testing on TC asap

File: regions/us-ca-ventura.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            33.94222067051576
+          ]
+        ]
+      ]
+    }
+  },
+  "currency": "USD",
+  "trafficRule": "right",
+  "feeds": [
+    "us-vctc"
```
[the remark this answers is not in the document]
Comment (fersagui) [reply regions/us-ca-ventura.json:71]: and this one: https://www.transit.land/feeds/f-santa~clarita/

File: regions/us-ca-ventura.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            34.48957975202644
+          ],
+          [
+            -119.81002807617188,
+            34.48957975202644
+          ],
+          [
+            -119.81002807617188,
+            33.94222067051576
+          ]
```
Comment (fersagui) [inline regions/us-ca-ventura.json:63]: This is touching LA polygon
Screen Shot 2022-10-25 at 08 54 49. We have to be sure the area is covered well.

**Review by fersagui — CHANGES_REQUESTED** [review]
Add any other operator which is in the area

File: regions/us-ca-ventura.json (hunk 2 above)
Comment (ManuBenito259) [reply regions/us-ca-ventura.json:63]: VCTC has routes going into Thousand Acres which is inside the LA polygon. The routes would be ecluded if the polygon is reduced, is that still correct?
imagen

Comment (nighthawk) [reply regions/us-ca-ventura.json:63]: It's a good opportunity to make the LA region a little bit smaller by moving Thousand Oaks and Simi Valley into the new Ventura region.

> VCTC has routes going into Thousand Acres which is inside the LA polygon. The routes would be ecluded if the polygon is reduced, is that still correct?

No, they will not b excluded. Routes can extend outside a region's polygon into another region's. They will then be included in both regions.

File: regions/us-ca-ventura.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (nighthawk) [reply regions/us-ca-ventura.json]: Should probably also include the existing Thousand Oaks feed, too.

File: regions/us-ca-ventura.json (hunk 2 above)
Comment (fersagui) [reply regions/us-ca-ventura.json:63]: @nighthawk what about this configuration? Screen Shot 2022-10-27 at 20 55 26

Comment (nighthawk) [reply regions/us-ca-ventura.json:70]: Looks good to me

**Review by fersagui — APPROVED** [review]
Polygon is now fixed and new sources added

**Pull request merged**